src/build/build.jl

```diff
@@ -47,10 +47,11 @@ end
     read_set(path::String)
 """
 function read_set(path::String)
-    cp(joinpath(SLIDE_DIR,"temp","gdpcat.csv"), joinpath(SLIDE_DIR,"data","coresets","gdpcat.csv"), force = true)
-    cp(joinpath(SLIDE_DIR,"temp","oth_use.csv"), joinpath(SLIDE_DIR,"data","coresets","i","oth_use.csv"), force = true)
-    cp(joinpath(SLIDE_DIR,"temp","cross_fd.csv"), joinpath(SLIDE_DIR,"data","coremaps","crosswalk","fd.csv"), force = true)
-    cp(joinpath(SLIDE_DIR,"temp","parse_fd.csv"), joinpath(SLIDE_DIR,"data","coremaps","parse","fd.csv"), force = true)
+    temp_dir = joinpath(SLIDE_DIR,"src","readfiles","_sets")
+    cp(joinpath(temp_dir,"gdpcat.csv"), joinpath(SLIDE_DIR,"data","coresets","gdpcat.csv"), force = true)
+    cp(joinpath(temp_dir,"oth_use.csv"), joinpath(SLIDE_DIR,"data","coresets","i","oth_use.csv"), force = true)
+    cp(joinpath(temp_dir,"cross_fd.csv"), joinpath(SLIDE_DIR,"data","coremaps","crosswalk","fd.csv"), force = true)
+    cp(joinpath(temp_dir,"parse_fd.csv"), joinpath(SLIDE_DIR,"data","coremaps","parse","fd.csv"), force = true)
```

Comment:
is there a better name than temp_dir here? set_dir? map_dir? setmap_dir?

Reply:
Absolutely!! This file is really old. I wanted to update master just so it would run, but in our newer release, these files are part of `./data/coresets/`/`data/coremaps`. This was sloppy, for sure. Thanks for calling me out on this!